gensim/models/coherencemodel.py

```diff
@@ -215,7 +218,7 @@ def __init__(self, model=None, topics=None, texts=None, corpus=None, dictionary=
         self._topics = None
         self.topics = topics
 
-        self.processes = processes if processes > 1 else max(1, mp.cpu_count() - 1)
+        self.processes = processes if processes >= 1 else max(1, mp.cpu_count() - 1)
```

Comment:
why?

Reply:
Looks like misclick, will be fixed in next commit.